src/confluent_kafka/admin/_group.py

```diff
@@ -123,14 +126,16 @@ class ConsumerGroupDescription:
         Partition assignor.
     state : ConsumerGroupState
         Current state of the consumer group.
+    type  : ConsumerGroupType
+        Type of the consumer group.
     coordinator: Node
         Consumer group coordinator.
     authorized_operations: list(AclOperation)
         AclOperations allowed for the consumer group.
     """
 
     def __init__(self, group_id, is_simple_consumer_group, members, partition_assignor, state,
-                 coordinator, authorized_operations=None):
+                 type, coordinator, authorized_operations=None):
```

Comment:
Don't change API contract.

```suggestion
                 coordinator, authorized_operations=None, type=ConsumerGroupType.UNKNOWN):
```